src/ErrorNotification.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useState } from 'react';
+import classnames from 'classnames';
+
+type Props = {
+  error: string;
+};
+
+export const ErrorNotification:React.FC<Props> = ({ error }) => {
+  const [hidden, setHidden] = useState(false);
+
+  const autoHideError = useCallback(
+    () => {
+      setTimeout(() => {
+        setHidden(true);
```

Comment:
There is no clearTimeout for the timeout set in autoHideError. This can lead to memory leaks when the component is unmounted before the timeout is completed. Use useEffect to clean up the timeout when the component is unmounted or the error prop changes.